apps/hefty/lib/hefty/trades.ex

```diff
@@ -166,4 +166,27 @@ defmodule Hefty.Trades do
   end
 
   defp update_profit(trade, _), do: trade
-end
+  
+  def count_gaining_losing(from, to) do
+    query =
+      "SELECT symbol, (cast(profit_base_currency as double precision) > 0) AS gaining, " <>
+      "COUNT(*) FROM trades " <>
+      "WHERE sell_time >= #{from} AND " <>
+      "sell_time < #{to} " <>
+      "GROUP BY (symbol, cast(profit_base_currency as double precision) > 0);"
+
+    Ecto.Adapters.SQL.query!(Hefty.Repo, query).rows
+      |> Enum.group_by(fn [head | _tail] -> head end)
+      |> Enum.map(fn {symbol, [[_, _, losing], [_, _, gaining]]} -> %{String.to_atom("#{symbol}") => [gaining, losing]} end)
```

Comment:
Instead of using `String.to_atom("#{symbol}")` it's popular to use `:"#{symbol}"`